api/envoy/api/v2/cds.proto

```diff
@@ -175,6 +175,15 @@ message Cluster {
     // specific load balancer. Consult the configured cluster's documentation for whether to set
     // this option or not.
     CLUSTER_PROVIDED = 6;
+
+    // [#not-implemented-hide:] Use the new :ref:`load_balancing_policy
```

Comment:
Very tiny: in Envoy, we have 100 char line length.

Reply:
Done.